metricflow/model/model_validator.py

```diff
@@ -47,13 +47,21 @@ class ModelValidator:
         ValidMaterializationRule(),
     ]
 
+    def __init__(self, rules: List[ModelValidationRule] = DEFAULT_RULES) -> None:
```

Comment:
.... and then this takes a `Tuple[ModelValidationRule, ...]`, or maybe a `Sequence[ModelValidationRule]`